src/pymatgen/optimization/linear_assignment.pyx

```diff
@@ -50,7 +50,8 @@ class LinearAssignment:
     """
 
     def __init__(self, costs, epsilon=1e-13):
-        self.orig_c = np.array(costs, dtype=np.float_, copy=False, order="C")
+        # https://numpy.org/devdocs/numpy_2_0_migration_guide.html#adapting-to-changes-in-the-copy-keyword
```

Comment:
@janosh This doesn't look like something we should keep inside the code (it's pretty easy to find)? What do you think?

Reply:
i'm generally in favor of linking relevant docs. even if easy to find, people might not think to look for them. but no strong opinion in this case

Reply:
This one looks more like a "disposable" migration guide which we would only do once :)